docs/source/guide/migrationec2.rst

```diff
@@ -0,0 +1,123 @@
+.. _guide_migration_ec2:
+
+Amazon EC2
+==========
+Boto 2.x contains a number of customizations to make working with Amazon EC2 instances, storage and networks easy.
```

Comment:
I would also include something about the benefit of using Boto3 over Boto2 specifically for EC2. It seems sort of weird that the opening statement is in short talking about how it is easy to use Boto2 for EC2 without any mention of Boto3 when this is a migration guide.